### PR TITLE
ISSUE #2197 - Fix clickable URL in comments section

### LINKED ISSUE #2197: URL no longer clickable in comments
### Description
url links in comments no longer turn into hyperlinks automatically

### Steps to replicate
- create a comment with a URL message
- Save comment
- The comment should highlight in blue and be clickable, but it remains to be plain text.

### Pictures/ Gifs of the error
image

### PR DESCRIPTION
**Pull request by MajaZubko**
This fixes #2197

### CODE REVIEW

**Comment by m1chalkubiak** [comment]
@MajaZubko Please fix lint errors

**Review by m1chalkubiak — CHANGES_REQUESTED** [review]
@MajaZubko Please fix lint errors

**Review by m1chalkubiak — DISMISSED** [review]

**Comment by MajaZubko** [comment]
@carmenfan this fix would be useful for us in other issues. Can you please look at it?

File: frontend/package.json [hunk 1]
```diff
@@ -55,6 +55,7 @@
     "history": "4.9.0",
     "konva": "4.0.9",
     "lodash": "4.17.15",
+    "markdown-linkify": "^1.0.1",
```
Comment (carmenfan) [inline frontend/package.json:58]: lock down the version please

We've had plenty of troubles in the past where a later version of the library changed the API and randomly broke deployments 😢

**Review by carmenfan — CHANGES_REQUESTED** [review]
- [ ] The link should spawn a new tab instead of going to the page directly.

**Review by carmenfan — APPROVED** [review]
Looks fine now!

@MajaZubko In the future please avoid rewriting history and altering a pre-existing commit. Just make a new commit!

It's not too big of a deal with a couple of line changes like this one, but if we are doing a 2nd review it's much faster to just see what you've changed base on what we've reviewed last time, instead of going through all the file changes again to look for a specific change. :)

**Pull request merged**